### PR TITLE
Added configuration option for SSL-handshake and Backend-connect timeout

### PR DESCRIPTION
**Pull request by dmatetelki**
The rebased & doc edited version of a deleted users's PR.

### CODE REVIEW

**Comment by dmatetelki** [comment]
"timeout" has inconsistent usages as options:

ocsp-resp-**tmo**
ocsp-connect-**tmo**
backend-connection-**timeout**
ssl-handshake-**timeout**

But that can be a separate commit.

**Comment by Dridi** [comment]
I see those inconsistencies in the oldcfg test, maybe we actually don't want to break this to maintain compatibility.

@daghf, opinion?

**Comment by dmatetelki** [comment]
The consensus was to create aliases and print warnings on the deprecated **tmo** usages.

**Comment by dmatetelki** [comment]
One final question. 

- `ocsp-resp-timeout` and
- `ocsp-connect-timeout`

are not available via command line. Shall I add that too?

**Comment by Dridi** [comment]
Show of hands in one hour ;)

File: CHANGES.rst [hunk 1]
```diff
@@ -4,6 +4,14 @@ List of changes
 This file contains the running log of changes applied to each released hitch
 version.
 
+hitch-1.4.8 (unreleased)
+------------------------
+
+* All options are now in sync and awailable via command line, config file and
```
Comment (klenan) [inline CHANGES.rst]: s,awail,avail,

Comment (dmatetelki) [reply CHANGES.rst]: Thank you!

**Pull request closed without merging**